### ISSUE TITLE
UI : Blind slider issue on dark theme

### ISSUE THREAD

**Issue by rezzalex** [domoticz/domoticz#5530]
With Google Chrome and Internet Explorer, and DZ latest version, (build 14928) the blind slider strikes through the rest of the Blind UI : 

![image](https://user-images.githubusercontent.com/35136439/211536016-faae6c25-1874-43f9-8b6c-825d1367bef5.png)

![image](https://user-images.githubusercontent.com/35136439/211536070-13b177a8-d34a-4b45-a60e-d819f578278b.png)

![image](https://user-images.githubusercontent.com/35136439/211536123-0e2bf4d6-c11e-4a35-9fc4-a08162ce0f05.png)

Would it be possible to hide it ? In my case, my blinds do not respond to the slider.

**Comment by gizmocuz** [comment]
How about with the default theme? And what theme are you using?

**Comment by rezzalex** [comment]
Fine with the default theme but I would prefer to user the element dark one.

with the dark theme : 
![image](https://user-images.githubusercontent.com/35136439/211539334-01fc15d0-45ed-4521-9a39-e77b57a7ffe6.png)

**Comment by gizmocuz** [comment]
Okey, then we know someone should correct the themes as the default is correct.
It's a matter of fixing this in the www/styles folder, please try to correct it (and clear the browser cache after each change to be sure) and make a PR or tell us what needs to be changed

**Comment by rezzalex** [comment]
ok, will try.

Would it be possible to remove the slider with a checkbox option ?

**Comment by gizmocuz** [comment]
Nope, but if you cannot position it, you can change it's type top 'blinds'

**Comment by rezzalex** [comment]
But I need the "Stop" button

**Comment by kofec** [comment]
Is it what you are looking for:
![image](https://user-images.githubusercontent.com/18610805/211546954-cc20918c-7b84-40d3-9692-40a41ee6ac81.png)
change type to: Venetian blind EU or US

**Comment by rezzalex** [comment]
Thank you @kofec , that's fine!
I though the recent blinds rework had removed those types of devices.
